pkg/vdri/bloc/config/consortium.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package config
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/square/go-jose"
+)
+
+/*
+A consortium config file is a concatenated JSON file: The first element is a JSON object containing:
+
+- The domain name of the consortium
+- Consortium policy configuration settings
+- A list of stakeholders - containing, for each stakeholder:
+  - The web domain where its configuration can be found
+  - The did:bloc DID of the stakeholder, with the associated DID doc in Sidetree on the consortium ledger
+-  The hash of the previous version of this config file
+After that, in order:
+-  A list of detached JWS signatures, from the stakeholders, on the config file
+-  A list of cached copies of the stakeholder config files
+*/
+
+// Consortium holds the configuration for a consortium, which is signed by stakeholders
+type Consortium struct {
+	// Domain is the domain name of the consortium
+	Domain string `json:"domain,omitempty"`
+	// Policy contains the consortium policy configuration
+	Policy ConsortiumPolicy `json:"policy"`
+	// Stakeholders is a list containing references to the stakeholders on this consortium
+	Stakeholders []StakeholderListElement `json:"stakeholders"`
+	// Previous contains a hashlink to the previous version of this file. Optional.
+	Previous string `json:"previous,omitempty"`
+}
+
+// Copy creates a deep copy of the Consortium
+func (cc *Consortium) Copy() *Consortium {
```

Comment:
Not sure why you need a Copy()?

Reply:
When I was working on my POC a week ago, it was useful. But yes, I don't think it's needed at all on the resolver side.

Reply:
I'll get rid of them.